src/cli/create-component.ts

```diff
@@ -20,8 +20,15 @@ const createDocumentationTitle = (component: string): string => {
 
 const capitalize = (sentence: string): string => sentence.length > 1 ? sentence.charAt(0).toUpperCase() + sentence.slice(1) : sentence.toUpperCase();
 
-const createMixin = (componentDirectory: string, component: string, prefix?: string): void => {
-  const content = `mixin ${dashPrefix(prefix)}${component}\n  .${dashPrefix(prefix)}${component} ${component}\n`;
+const createMixin = (componentDirectory: string, component: string, prefix?: string, mixinOptions?: boolean): void => {
+  let content = `mixin ${dashPrefix(prefix)}${component}\n  .${dashPrefix(prefix)}${component} ${component}\n`;
+  if(mixinOptions) {
+    content = `mixin ${dashPrefix(prefix)}${component}(options)\n` +
```

Comment:
maybe you can extract to a function that returns content in order to prevent `content` mutation here